go.mod

```diff
@@ -1,23 +1,25 @@
 module github.com/ibm/varnish-operator
 
-go 1.13
+go 1.15
 
 require (
-	github.com/caarlos0/env/v6 v6.2.1
+	github.com/caarlos0/env/v6 v6.5.0
 	github.com/docker/distribution v2.7.1+incompatible
-	github.com/go-logr/zapr v0.1.0
-	github.com/gogo/protobuf v1.3.1
-	github.com/google/go-cmp v0.3.1
-	github.com/onsi/ginkgo v1.11.0
-	github.com/onsi/gomega v1.8.1
-	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
-	github.com/pkg/errors v0.8.1
-	github.com/prometheus/client_golang v1.2.1
+	github.com/go-logr/logr v0.4.0 // indirect
+	github.com/go-logr/zapr v0.4.0
+	github.com/gogo/protobuf v1.3.2
+	github.com/google/go-cmp v0.5.4
+	github.com/onsi/ginkgo v1.15.0
+	github.com/onsi/gomega v1.10.5
+	github.com/opencontainers/go-digest v1.0.0 // indirect
+	github.com/pkg/errors v0.9.1
+	github.com/prometheus/client_golang v1.9.0
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.7.0
-	go.uber.org/zap v1.10.0
-	k8s.io/api v0.17.4
-	k8s.io/apimachinery v0.17.4
-	k8s.io/client-go v0.17.4
-	sigs.k8s.io/controller-runtime v0.5.2
+	github.com/prometheus/common v0.17.0
+	go.uber.org/zap v1.16.0
+	k8s.io/api v0.20.4
+	k8s.io/apimachinery v0.20.4
+	k8s.io/client-go v0.20.4
+	sigs.k8s.io/controller-runtime v0.8.2
+	sigs.k8s.io/controller-tools v0.5.0 // indirect
```

Comment:
that line doesn't seem right. We shouldn't need the `controller-tools` to be imported.
could you run `go mod tidy` to see if it disappears? That command removes not imported libraries and adds imported. Should fix it if it get there by accident.

Reply:
`go mod tidy` cleaned this one up. I'm not sure why, but it added another indirect import though. `k8s.io/apiextensions-apiserver v0.20.2`, which I bumped up to match all the other k8s imports (v0.20.4).